solr/core/src/java/org/apache/solr/api/ContainerPluginsRegistry.java

```diff
@@ -343,7 +343,12 @@ public ApiInfo(PluginMetaHolder infoHolder, List<String> errs) {
           return;
         }
       } else {
-        klas = coreContainer.getResourceLoader().findClass(klassInfo.className, Object.class);
+        try {
+          klas = coreContainer.getResourceLoader().findClass(klassInfo.className, Object.class);
+        } catch (Exception e) {
+          errs.add("Error loading class " + e.toString());
```

Comment:
Looking at the findClass code, I can see that it's exception messages are sufficiently clear.  Consequently, I think you needn't prefix by "Error loading class ".